sql/plan/alter_index.go

```diff
@@ -283,26 +290,74 @@ func (p *AlterIndex) RowIter(ctx *sql.Context, row sql.Row) (sql.RowIter, error)
 	return sql.RowsToRowIter(sql.NewRow(types.NewOkResult(0))), nil
 }
 
-// WithChildren implements the Node interface.
-func (p *AlterIndex) WithChildren(children ...sql.Node) (sql.Node, error) {
+// For plan.AlterIndex, one child is expected - The table.
```

Comment:
docstrings usuallly follow the pattern of `// <name> does a thing/returns a thing/otherwise returns these types of errors`